server/api/tests/test_services.py

```diff
@@ -0,0 +1,89 @@
+import pytest
+from services import nc, map, requests, visualizations
+
+"""
+These are 'unit tests' of the key API services.
+
+However, these tests run on a live database and assume that data
+from the first 1/2 of 2020 have been loaded.
```

Comment:
haha. yep. i figured this out eventually after looking at the CI YAML. i was wondering why all my tests failed!

i'm currently working on an option to seed the DB from a file instead of by calling Socrata. i'm going to try to do something quick and dirty for now and then we can improve once we add Alembic

Reply:
perfect